docs/DeveloperGuide.md

```diff
@@ -270,58 +270,208 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* *`    | SoC professor                          | view the progress updates that my students upload                                                                                       | deliberate about what the next course of action should be [coming soon]       |
+| `* *`    | eager to learn student                 | receive feedback (read comments and suggestions) from my professor                                                                      | I can set a direction to make progress for my FYP [coming soon]               |
+| `* *`    | responsible student                    | update my FYP progress through the app                                                                                                  | professor can be informed on my progress [coming soon]                        |
+| `* *`    | critical student                       | provide feedbacks on my FYP guidance and evaluation                                                                                     | so that I can help the FYP management team to further improve [coming soon]   |
+| `* *`    | helpful SoC professor                  | put tags on my student's FYP project so that I can let him/her know whether they are “Ahead of time”, “Up to task” or “Behind schedule” | they can be informed of their progress relative to the timeline [coming soon] |
+| `* *`    | non-confrontational (i.e. shy) student | push certain tags (e.g. “Assistance required!”, “Unable to complete :(”) to the application                                             | gather my professor’s attention without direct confrontation [coming soon]    |
+| `*`      | fun-seeking professor/FYP student      | end “stickers” (similar to Telegram stickers) or emoji so that the app                                                                  | is more appealing to use [coming soon]                                        |
+| `*`      | FYP student                            | update my availability slots on the app                                                                                                 | my professor knows when I will be available to meet [coming soon]             |
+| `*`      | well-versed SoC professor              | sort my student’s FYP based on what specialisation they are taking                                                                      | I can better organise the FYPs [coming soon]                                  |
+| `*`      | very meticulous SoC professor          | see student's FYP sorted by research area first, followed by alphabetical order                                                         | I can gain quick easy access to the details required [coming soon]            |
 
 *{More to be added}*
 
+--------------------------------------------------------------------------------------------------------------------
```

Comment:
If anyone has other idea to make each Use Case more distnct, do suggest

Reply:
I was thinking to put these lines in between each UC instead of just before the first UC and just after the last UC

Reply:
Were we taught any other ways to differenciate the points? Maybe we can also underline the UCs? If it's not prominent enough we can also use the line between each UC, just that might look alittle confusing since the lines currently represent different segments of the docs (eg:User Stories, User Guide).

Reply:
Not really, so it's just cosmetic reasons. You can ignore this then I suppose.